spark/common/src/test/scala/org/apache/sedona/sql/functions/STS2CellIDs.scala

```diff
@@ -100,4 +100,17 @@ class STS2CellIDs extends TestBaseScala with Matchers with GeometrySample with G
       )
     }
   }
+
+  describe("should pass ST_S2ToGeom") {
+    it("should intersect with the target geometry") {
```

Comment:
Please add one more test to show that the result of ST_S2ToGeom can be used in ST_Collect to get a `MULTIPOLYGON`